sdk/tables/azure-data-tables/inc/azure/data/tables/rest_client.hpp

```diff
@@ -186,7 +186,7 @@ namespace Azure { namespace Data { namespace Tables {
      */
     explicit TableClient(
         const std::string& tableName,
-        std::shared_ptr<StorageSharedKeyCredential> credential,
+        std::shared_ptr<Azure::Storage::StorageSharedKeyCredential> credential,
```

Comment:
Thanks! This should work.
BTW, if you don't like the long namespaces, you could write as just "Storage::" here, it will work too.

Reply:
i kind like the complete namspace